projects/aws/image-builder/builder/builder.go

```diff
@@ -154,13 +154,33 @@ func (b *BuildOptions) BuildImage() {
 		}
 
 		outputArtifactPath = filepath.Join(cwd, fmt.Sprintf("%s.gz", b.Os))
+	} else if b.Hypervisor == NutanixAHV {
+		// Read and set the nutanix connection data
+		nutanixAHVConfigData, err := json.Marshal(b.NutanixAHVConfig)
+		if err != nil {
+			log.Fatalf("Error marshalling nutanix ahv config data")
+		}
+		err = ioutil.WriteFile(filepath.Join(upstreamImageBuilderProjectPath, "packer/nutanix/nutanix.json"), nutanixAHVConfigData, 0644)
```

Comment:
Is this replacing the upstream config file?

Reply:
yes, as we want to pass the variables given in a file by user in the --nutanix-config parameter of aws image-builder cli